sdk/keyvault/azure-security-keyvault-certificates/README.md

```diff
@@ -277,22 +277,22 @@ You will notice that additional information is logged, like the client request I
 
 Several Azure Key Vault secrets client library samples are available to you in this GitHub repository. These samples provide example code for additional scenarios commonly encountered while working with Azure Key Vault:
 
-* [Certificate-Basic-Operations](https://github.com/Azure/azure-sdk-for-cpp/tree/main/sdk/keyvault/azure-security-keyvault-certificates/samples/certificate-basic-operations):
```

Comment:
Why remove the links?

Reply:
Makes CI to fail on link verification. I will re-enable them after merging. 
See the fail: https://dev.azure.com/azure-sdk/public/_build/results?buildId=1196363&view=logs&j=a129effc-2dd1-54d1-fb5a-ad7bdc0e851d&t=7b4713e3-e110-5049-e231-cc8a1e664c68&l=42

It seams like the link verifier can now check the links that are suppose to be in main within the same branch.  I can make a test updating the link to see if it works in this same PR.